parcels/tools/_helpers.py

```diff
@@ -74,7 +74,6 @@ def field_repr(field: Field) -> str:
     name            : {field.name!r}
     grid            : {field.grid!r}
     extrapolate time: {field.allow_time_extrapolation!r}
-    time_periodic   : {field.time_periodic!r}
```

Comment:
There are still a lot of notebook outputs that show this time_periodic. But perhaps we can better wait until the end of the v4-development to update/rerun all the notebooks?

Reply:
yep, that was what I was thinking as well